src/sql/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -49,3 +81,24 @@ CommandsRegistry.registerCommand({
 		}
 	}
 });
+export class ADSWebLite implements IWorkbenchContribution {
+	constructor(
+		@ICommandService private commandService: ICommandService,
+	) {
+		this.registerEditorOverride();
+	}
+
+	private async registerEditorOverride(): Promise<void> {
+		if (product.quality === 'tsgops-image') {
```

Comment:
Please define the quality as a const in a common location.

Is there a reason you went for tsgops-image instead of just tsgops or something similar? Not a huge deal but seems unnecessarry to have it be that long